tests/integration/testdata/invoke/template-pseudo-params.yaml

```diff
@@ -0,0 +1,67 @@
+AWSTemplateFormatVersion: '2010-09-09'
+Transform: AWS::Serverless-2016-10-31
+Description: Run intrinsics and fill in pseudo parameters first.
+
+Mappings:
+  MappingExample:
+    eu-west-1:
+      ApplicationId: arn:aws:serverlessrepo:us-east-1:077246666028:applications/hello-world
```

Comment:
Can we use the `1234...` fake account here?

Reply:
sure

Reply:
turns out we cant, the serverless app plugin makes a call to serverless application repo based on the applicationId.